httpcore5-testing/src/main/java/org/apache/hc/core5/testing/classic/LoggingHttp1StreamListener.java

```diff
@@ -52,9 +52,9 @@ public void onResponseHead(final HttpConnection connection, final HttpResponse r
     public void onExchangeComplete(final HttpConnection connection, final boolean keepAlive) {
         if (connLog.isDebugEnabled()) {
             if (keepAlive) {
-                connLog.debug(LoggingSupport.getId(connection) + " Connection is kept alive");
+                connLog.debug("{} Connection is kept alive", LoggingSupport.getId(connection));
             } else {
-                connLog.debug(LoggingSupport.getId(connection) + " Connection is not kept alive");
+                connLog.debug("{} Connection is not kept alive", LoggingSupport.getId(connection));
```

Comment:
Other messages with id use a colon to separate. If it is such a message use a colon otherwise lowercase connection.